docs/framework/deployment/deployment-guide-for-developers.md

```diff
@@ -12,9 +12,9 @@ This topic provides information for developers who want to install any version o
 
 For download links, see the section [Redistributable Packages](#redistributable-packages). You can also download the redistributable packages and language packs from these Microsoft Download Center pages:
```

Comment:
I'd keep this file out of this effort for now. There are still some download links being converted on this one.